src/legacy/core_plugins/metrics/public/visualizations/components/top_n.js

```diff
@@ -59,14 +60,21 @@ class TopN extends Component {
           onClick={this.handleClick({ lastValue, ...item })}
           style={styles.row}
         >
-          <td width="1*" className="tvbVisTopN__label">{ item.label }</td>
+          <td className="tvbVisTopN__label">
+            <EuiToolTip
+              position="right"
+              content={item.label}
+            >
+              <span>{ item.label }</span>
```

Comment:
Do we really need this span? Looked just fine for me without spans and the max-width on the td and a title attribute on the td as @cchaos suggested.
<img width="379" alt="screenshot 2019-02-20 at 16 14 54" src="https://user-images.githubusercontent.com/1508364/53101763-b00c3780-352a-11e9-8d90-4187a422e4ad.png">

Reply:
Yep, you are completely right! Span is not necessary here, it was needed when there was a `EuiToolTip` component which needs a component as a child